docs/handbook/getting-started.md

```diff
@@ -1937,8 +1959,9 @@ Nearly all cards can use the VESA driver, and are therefore supported for 2D.
 
 ### Video card support (3D)
 
-* Beginning with the July 2016 experimental release, both Nvidia and Intel GPU's are now supported.
-* AMD GPU's are not currently supported (VESA driver only).
+* Nearly all NVidia GPU's are supported by the Nvidia binary driver.
+* Beginning with the July 2016 experimental release, most Intel GPU's are now supported.
+* While AMD GPU's are not currently supported (VESA driver only), development is currently underway to provide 3D support for both legacy and modern AMD GPU's.
```

Comment:
@pyhalov , How does this sound?

### Video card support (2D)

* Nearly all cards can use the VESA driver, and are therefore supported for 2D.
* 2D acceleration for NVidia GPU's is provided by the Nvidia binary driver.
* Work has begun to provide Radeon (2D) acceleration support for both legacy and modern AMD GPU's.

### Video card support (3D)

* Nearly all NVidia GPU's are supported by the Nvidia binary driver.
* Beginning with the July 2016 experimental release, most Intel GPU's are now also supported.
* No 3D support is currently available for AMD GPU's.